### PR TITLE
[PDI-20251] adding missing log4j libraries

### PR DESCRIPTION
**Pull request by srallapa**

### CODE REVIEW

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>



---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **334**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi-dataservice-plugin/job/PR-223/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <pdi.version>10.3.0.0-SNAPSHOT</pdi.version>
     <pentaho-metastore.version>10.3.0.0-SNAPSHOT</pentaho-metastore.version>
     <rxjava.version>2.2.3</rxjava.version>
+    <log4j.version>2.24.1</log4j.version>
```
Comment (ddiroma) [inline pom.xml]: @srallapa log4j version is set in maven-parent-poms. Please determine if we need to overwrite that version here

Comment (srallapa) [reply pom.xml]: not required, I am removing these changes and keeping only assembly changes

Comment (srallapa) [reply pom.xml]: updated.

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>



---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 41s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **334**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi-dataservice-plugin/job/PR-223/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>



---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl assemblies/driver
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**